package.json

```diff
@@ -50,7 +50,7 @@
     "core-object": "^1.1.0",
     "ember-cli-babel": "^5.0.0",
     "ember-cli-deploy-plugin": "^0.2.2",
-    "lodash": "^3.9.3",
+    "lodash": "^4.11.1",
```

Comment:
@YoranBrondsema why are we updating this?

I ask because lodash has been giving us quite a bit of gas lately in different plugins and I've seen some weird npm incompatibilities around it